packages/extension-browser/src/devtools/utils/storage.ts

```diff
@@ -1,6 +1,30 @@
+let storage: {
+    getItem(key: string): string | null;
+    setItem(key: string, value: string): void;
+};
+
+try {
+    storage = {
+        getItem: localStorage.getItem.bind(localStorage),
+        setItem: localStorage.setItem.bind(localStorage)
+    };
+} catch (error) {
+    console.warn(`LocalStorage not available, falling back to in-memory storage: ${error}`);
+    const cache = new Map<string, string>();
```

Comment:
We'll know for sure once CI finishes, but I think our linter is configured to require a new line before `const`.
```suggestion

    const cache = new Map<string, string>();
```

Reply:
ESLint doesn't complain and it doesn't seem to need one here:
https://github.com/webhintio/hint/blob/98861bac7336d535ae100d680497a01e9ad3113a/packages/extension-browser/src/devtools/utils/storage.ts#L2-L3

Reply:
CI passed so I must have been thinking of a different rule 🤷‍♂️

Reply:
Maybe you are thinking on `return`, the linter is configured you require a new line after `const`